sources/packages/backend/workflow/src/workflow-definitions/fulltime-assessment-2025-2026.bpmn

```diff
@@ -119,8 +119,9 @@ studentCalculatedTotalIncome</bpmn:text>
       <bpmn:text>IN
 studentDataRelationshipStatus
 calculatedDataHasEligibleDependants
-studentDataLivingWithParents
-studentDataLivingathomeRent
+studentDataLivingAtHome
+studentDataLivingAtHomeRent
```

Comment:
Is `studentDataLivingAtHomeRent` added to the DMN input?  I assume that it must be an input based on the comment.